security/cmd/istio_ca/main.go

```diff
@@ -211,8 +212,8 @@ func init() {
 		defaultWorkloadMinCertGracePeriod, "The minimum workload certificate rotation grace period.")
 
 	// gRPC server for signing CSRs.
-	flags.StringVar(&opts.grpcHostname, "grpc-hostname", "istio-citadel", "The hostname for GRPC server.")
-	flags.StringVar(&opts.grpcHosts, "grpc-host-identities", "istio-citadel",
+	flags.StringVar(&opts.grpcHostname, "grpc-hostname", "istio-ca", "DEPRECATED, use --grpc-host-identites.")
+	flags.StringVar(&opts.grpcHosts, "grpc-host-identities", "istio-ca",
```

Comment:
Since we already support multiple host identities, can we just put both `istio-ca` and `istio-citadel` here?

Reply:
I'll do that in a follow-up PR to expose istio-citdael as well. So that I'll manually test it to make sure it works.

Reply:
OK. Thanks.

Reply:
Changed.

Per offline discussion, we need to include both istio-ca and istio-citadel since the yaml file for 
service config has been updated. dnsmasq deb package is still istio-ca. In short, we'll provide three service names for the ca identites:
1. istio-ca (for node agent in vm expansion)
1. istio-citadel (ilb has been updated to istio-citdadel)
1. istio-citadel.istio-system.svc.cluster.local (for node agent in k8s)